app/lib/features/search/pages/quick_search_page.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/widgets/acter_search_widget.dart';
+import 'package:acter/features/pins/widgets/pin_list_widget.dart';
+import 'package:acter/features/search/providers/quick_search_providers.dart';
+import 'package:acter/features/spaces/widgets/space_list_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:go_router/go_router.dart';
+
+class QuickSearchPage extends ConsumerStatefulWidget {
+  const QuickSearchPage({super.key});
+
+  @override
+  ConsumerState<QuickSearchPage> createState() => _QuickSearchPageState();
+}
+
+class _QuickSearchPageState extends ConsumerState<QuickSearchPage> {
+  String get searchValue => ref.watch(quickSearchValueProvider);
```

Comment:
So, I like this approach of watching states over ```build()```, IF IT DOESN'T come with side-effects. Can we document this in dev docs recommended practices to follow in all places ?